src/common/components/footer/footer.module.scss

```diff
@@ -6,7 +6,8 @@
   grid-column: 1 / -1 !important;
 
   // koros stuff
-  margin-top: -1rem;
+  margin-top: -14px;
```

Comment:
How were you able to determine this value? is it something we could document here?

Reply:
It's the height of koros

Reply:
-0.9em works the same, but since the koros was defined with pixels, I used them.